source/get-started/download-and-install.txt

```diff
@@ -0,0 +1,84 @@
+.. _kotlin-sync-download-install:
+
+====================
+Download and Install
+====================
+
+.. procedure::
+   :style: connected
+
+   .. step:: Create a {+language+} Project
+
+      First, make sure that your system has {+language+} installed and
+      running on JDK 1.8 or later.
+
+      This guide shows you how to create a project and add the
+      {+driver-short+} dependencies by using Gradle or Maven. We
+      recommend that you use an integrated development
+      environment (IDE) such as IntelliJ IDEA or Eclipse IDE to
+      configure Gradle or Maven to build and run your project.
```

Comment:
I'd still lead with the "This guide..." sentence, maybe pull it up to the top level before the steps:
```suggestion

This guide shows you how to create a project and add the
{+driver-short+} dependencies by using Gradle or Maven. 

.. procedure::
   :style: connected

   .. step:: Create a {+language+} Project

      First, make sure that your system has {+language+} installed and
      running on JDK 1.8 or later.

      We recommend that you use an integrated development
      environment (IDE) such as IntelliJ IDEA or Eclipse IDE to
      configure Gradle or Maven to build and run your project.
```